packages/components/src/internal/components/assay/AssayImportPanels.tsx

```diff
@@ -276,6 +282,63 @@ class AssayImportPanelsBody extends Component<Props, State> {
         return columns;
     };
 
+    getRunColumns = (
+        runQueryInfo: QueryInfo
+    ): { plateColumns: OrderedMap<string, QueryColumn>; runColumns: OrderedMap<string, QueryColumn> } => {
+        const runColumns = this.getDomainColumns(AssayDomainTypes.RUN, runQueryInfo).asMutable();
+        const plateColumns = OrderedMap<string, QueryColumn>().asMutable();
+
+        if (this.plateSupportEnabled) {
+            const plateColumnFieldKeys = [PLATE_TEMPLATE_COLUMN];
+
+            plateColumnFieldKeys
+                .map(fieldKey => fieldKey.toLowerCase())
+                .forEach(fieldKey => {
```

Comment:
I'm a little confused by this. Is this usage of a `plateColumnFieldKeys` array just to future proof this code for when more columns are added? Currently, there is just one, so it seems like the map and forEach aren't necessary and just the if statement below is what is needed.

Reply:
I thought I'd be reusing this loop within this PR. Removed the looping.